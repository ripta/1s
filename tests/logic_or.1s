"lib/logic.1s" {LOAD}

"not"
#true !
#false !

"and"
#false #false &&
#false #true &&
#true #false &&
#true #true &&

"inclusive_or"
#false #false ||
#false #true ||
#true #false ||
#true #true ||

"exclusive_or"
#false #false ^^
#false #true ^^
#true #false ^^
#true #true ^^
