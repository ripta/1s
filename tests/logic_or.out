Result:
  "not"
  SymbolU32 { value: 10 }
  SymbolU32 { value: 11 }
  "and"
  SymbolU32 { value: 10 }
  SymbolU32 { value: 10 }
  SymbolU32 { value: 10 }
  SymbolU32 { value: 11 }
  "inclusive_or"
  SymbolU32 { value: 10 }
  SymbolU32 { value: 11 }
  SymbolU32 { value: 11 }
  SymbolU32 { value: 11 }
  "exclusive_or"
  SymbolU32 { value: 10 }
  SymbolU32 { value: 11 }
  SymbolU32 { value: 11 }
  SymbolU32 { value: 10 }
